zanata-model/src/main/java/org/zanata/model/HAccount.java

```diff
@@ -58,7 +58,7 @@
  *
  */
 @Entity
-@Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)
```

Comment:
I'm not sure if we really need to be specifying `@Cache(usage = CacheConcurrencyStrategy.TRANSACTIONAL)` everywhere, but if so we've probably missed a few because this code was sidelined for a year.